### PR TITLE
3 new bounty missions for lower level characters with interceptors and light warships.

### PR DESCRIPTION
**Pull request by samoja12**
**Content (Missions )**

## Summary
{{Added 3 new bounty missions to jobs.txt, mainly intended for early game play, thay feature light interceptors, heavy interceptors and light warships, with appropriate rewards. Lowest level mission starts at 50 CR, though i was debating making it have no CR requirement to allow for early combat start.})

### CODE REVIEW

**Review by arkhne — APPROVED** [review]
I see no overt problems, and I see no reason to not include these changes.

Also, welcome to the club.

**Comment by samoja12** [comment]
> 
> 
> I see no overt problems, and I see no reason to not include these changes.
> 
> Also, welcome to the club.

Thanks, nice to be here.

File: data/human/jobs.txt [hunk 1]
```diff
@@ -2575,8 +2575,144 @@ mission "Recycle garbage"
...
+				"Fury"
+			variant
+				"Fury (Missile)"
+			variant
+				"Fury (Heavy)"
+			variant
+				"Fury (Laser)"
+			variant
+				"Fury (Bomber)"
+
```
Comment (Anarchist2) [inline data/human/jobs.txt:2668]:
```suggestion
```

**Review by Anarchist2 — CHANGES_REQUESTED** [review]
Make sure to re-order the missions in ascending difficulty.

Also take note that Small bounty hunting uses `bandit` in the name, Medium uses `pirate` and Large uses `warlord`, while the new missions only use `pirate`.

File: data/human/jobs.txt [hunk 2]
```diff
@@ -2575,8 +2575,144 @@ mission "Recycle garbage"
...
+		dialog phrase "generic bounty hunting payment dialog"
+
+mission "Bounty Hunting (veryTiny)"
+	name "Wanted pirate near <system>"
+	description "A small pirate fighter named the <npc> has been attacking merchants near the <system> system. Destroy it and return to <planet> for payment (<payment>)."
+	repeat
+	job
+	to offer
+		"combat rating" > 50
+		random < 40
```
Comment (ravenshining) [inline data/human/jobs.txt:2683]: In addition to Anarchist's suggestion about limiting combat rating, you might gate it behind ship type, for example:

```suggestion
		"combat rating" < 500
		random < 40
		or
			has "ships: Interceptor"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			"combat rating" > 50
```

File: data/human/jobs.txt [hunk 3]
```diff
@@ -2575,8 +2575,144 @@ mission "Recycle garbage"
 		payment
 		payment 20000
 
+mission "Bounty Hunting (verySmall)"
+	name "Wanted pirate near <system>"
+	description "A light pirate warship named the <npc> has been attacking merchants near the <system> system. Destroy it and return to <planet> for payment (<payment>)."
+	repeat
+	job
+	to offer
+		"combat rating" > 200
```
[the remark this answers is not in the document]
Comment (samoja12) [reply data/human/jobs.txt:2584]: Was thinking about that, but i don't want to limit players in that way, maybe someone wants to do low end bounty missions with a KIV, i know i do from time to time, just to flex my big fleet. it should be ok.

File: data/human/jobs.txt [hunk 4]
```diff
@@ -2575,8 +2575,144 @@ mission "Recycle garbage"
...
+		dialog phrase "generic hunted bounty eliminated dialog"
+	on visit
+		dialog phrase "generic bounty hunting on visit"
+	on complete
+		payment 20000
+		dialog phrase "generic bounty hunting payment dialog"
+
 mission "Bounty Hunting (Small)"
 	name "Wanted bandit near <system>"
 	description "A low-end pirate warship named the <npc> has been attacking merchants near the <system> system. Destroy it and return to <planet> for payment (<payment>)."
```
[the remark this answers is not in the document]
Comment (samoja12) [reply data/human/jobs.txt:2718]: Not sure i want to change any of the existing missions without express blessing from the Admins.

File: data/human/jobs.txt [hunk 5]
```diff
@@ -2575,8 +2575,144 @@ mission "Recycle garbage"
 		payment
 		payment 20000
 
+mission "Bounty Hunting (verySmall)"
```
[the remark this answers is not in the document]
Comment (samoja12) [reply data/human/jobs.txt:2578]: Fixed, blame the coder in me.

File: data/human/jobs.txt (hunk 2 above)
Comment (samoja12) [reply data/human/jobs.txt:2682]: Nah, again, i don't want to limit players in that way.

File: data/human/jobs.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (samoja12) [reply data/human/jobs.txt:2578]: Fixed thanks

**Comment by samoja12** [comment]
There were some issues, i accidentally changed the master instead of creating a branch, i will reopen this pull request properly, please delete this, if possible.

File: data/human/jobs.txt (hunk 2 above)
Comment (ravenshining) [reply data/human/jobs.txt:2682]: The `has "ships: xxxx" lines would make the mission appear *more*, not less - enabling a player with a low rep but an appropriate ship to see the missions.

**Pull request closed without merging**